File: 14.LongestCommonPrefix/Solution.py
from typing import List

class Solution:
    def longestCommonPrefix(self, strs:List[str])->str:
        strs.sort()
        low,high = strs[0] , strs[len(strs)-1]
        i = 0
        while i < len(low) and low[i] == high[i]:
            i += 1
        return low[0:i]

File: 14.LongestCommonPrefix/Solution.java
import java.util.Arrays;
public class Solution{
    public String longestCommonPrefix(String[] strs) {
        Arrays.sort(strs);
        String low = strs[0], high = strs[strs.length - 1];
        int i = 0;
        while(i < low.length() && low.charAt(i) == high.charAt(i)) i++;
        return low.substring(0, i);
    }
}